packages/core/useContextMenu/index.ts

```diff
@@ -0,0 +1,34 @@
+
+import { computed, createVNode, ref, render } from 'vue-demi'
+import type { Fn } from '@vueuse/shared'
+import { unrefElement } from '../unrefElement'
+import { useEventListener } from '../useEventListener'
+import type { MaybeElementRef } from '../unrefElement'
+import { ContextMenu } from './component'
+
+export interface MenuItem {
+  text: string
+  handler: Fn
+}
+
+export interface MenuPosition {
+  x: Number
+  y: Number
+}
+
+export function useContextMenu(element: MaybeElementRef,
+  items: MenuItem[]) {
+  const target = computed(() => unrefElement(element))
+  const position = ref<MenuPosition>({ x: 0, y: 0 })
+
+  const menu = computed(() => createVNode(ContextMenu, { items, position }))
```

Comment:
Thanks but no. I don't think we should create vnodes in compsoables.

Reply:
Thanks a lot @antfu